lib/koans/17_map_sets.ex

```diff
@@ -0,0 +1,62 @@
+defmodule MapSets do
+  use Koans
+
+  @intro "My name is Set, MapSet"
+
+  @set MapSet.new([1, 2, 3, 4, 5])
+
+  koan "Does this value exist in the map set?" do
```

Comment:
I sort of feel like the first introduction to MapSet should be something like "It's a collection, like a list", and then the second lesson be "But it's different in that each element is distinct, like a set". What we have here gets off to a slow start and sort of makes me want to ask "wait what is this thing for?"

Does that make sense? Maybe I'm crazy? 😆